cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -351,32 +365,52 @@ func newMigrations(
 	dir string,
 	nWorker int, // number of concurrent worker to migation payloads
 	runMigrations bool,
+	fixSlabsWithBrokenReferences bool,
 ) []ledger.Migration {
 	if runMigrations {
+
 		rwf := reporters.NewReportFileWriterFactory(dir, log)
 
-		migrations := []ledger.Migration{
+		var accountBasedMigrations []migrators.AccountBasedMigration
+
+		if fixSlabsWithBrokenReferences {
+			accountBasedMigrations = append(
+				accountBasedMigrations,
+				migrators.NewFixBrokenReferencesInSlabsMigration(
+					rwf,
+					migrators.TestnetAccountsWithBrokenSlabReferences,
+				),
+			)
+		}
+
+		accountBasedMigrations = append(
+			accountBasedMigrations,
+			migrators.NewFixBrokenReferencesInSlabsMigration(
+				rwf,
+				migrators.TestnetAccountsWithBrokenSlabReferences,
+			),
```

Comment:
`FixBrokenReferencesInSlabsMigration` is added already under a flag, so we don't need it here.

```suggestion
```

Reply:
Ah, good catch!